packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -290,7 +280,6 @@ const pushTask = task =>
  */
 module.exports = ({
   url,
-  store,
   cache,
```

Comment:
we should also make it so we accept `getCache` function here, so we can grab cache for `gatsby-source-filesystem` instead of plugin's that called this function, optionally we could warn about using `cache` instead of `getCache` (but we can't terminate, because that would be breaking change - specifically for users that use plugins that didn't make swap)